docs/topics/functions.md

```diff
@@ -127,37 +121,32 @@ reformat(
 )
 ```
 
-You can skip all the ones with default values:
+You can disregard all of the ones with default values:
 
 ```kotlin
 reformat("This is a long String!")
 ```
 
-You are also able to skip specific arguments with default values, rather than omitting them all. However, after the first
-skipped argument, you must name all subsequent arguments:
+You can also skip specific arguments with default values rather than omitting all of them. After the first skipped argument, however, you must name all subsequent arguments:
 
 ```kotlin
 reformat("This is a short String!", upperCaseFirstLetter = false, wordSeparator = '_')
 ```
 
-You can pass a [variable number of arguments (`vararg`)](#variable-number-of-arguments-varargs) with names using the
-`spread` operator:
+Using the'spread' operator, you can pass a [variable number of arguments ('vararg')](#variable-number-of-arguments-varargs) with names:
```

Comment:
```suggestion
You can pass a [variable number of arguments (`vararg`)](#variable-number-of-arguments-varargs) with names using the `spread` operator:
```
We prefer to enclose operator names and variables in backticks to format them as inline code.